portal-user-models/src/main/java/com/telecominfraproject/wlan/portaluser/models/PortalUser.java

```diff
@@ -20,7 +21,7 @@
     //TODO: put more fields here, generate getters/setters for them
     private String username;
     private String password;
-    private PortalUserRole role;
+    private List<PortalUserRole> role;
```

Comment:
I think the new property name should be "roles", with accessors getRoles/setRoles - to better reflect the type.
I understand, it could break compatibility with existing deployments. 
What we can do is to have accessors for both of them, like this:

`
private List<PortalUserRole> roles;

public List<PortalUserRole> getRoles(){
  return roles;
}

public void setRoles(List<PortalUserRole> roles ){
  this.roles = roles;
}


@JsonIgnore
@Deprecated
public PortalUserRole getRole() {
  if(roles == null || roles.isEmpty()){
    return null;
  }
  return roles.get(0);
}

@JsonIgnore
@Deprecated
public void setRole(PortalUserRole role) {
  roles = new ArrayList<>();
  roles.add(role);
}
`
Sometime later we will remove deprecated methods getRole and setRole.